lib/cronitor/monitor.rb

```diff
@@ -0,0 +1,190 @@
+# frozen_string_literal: true
+
+def symbolize_keys(obj)
+  obj.inject({}){|memo,(k,v)| memo[k.to_sym] = v; memo}
+end
+
+module Cronitor
+  class Monitor
+    attr_reader :key, :data, :api_key, :api_version, :env
+
+    @@HEADERS = {
```

Comment:
Class variables are really a last resort; it’s why when I wrote this gem originally, I made it a class & used instance vars, instead of writing a module.

Might there be another way to do this? Or maybe it’s not necessary for it to be a class var at all?

Reply:
Looked through the HTTParty docs & code a bit, and I think it’d be fine to use a regular scope-level var here, instead of a class var:

```ruby
HEADERS = {
```

Reply:
ty!